omero/sysadmins/server-upgrade.rst

```diff
@@ -211,8 +211,8 @@ Upgrade your database
 .. only:: point_release
 
     .. warning::
-        This section only concerns users upgrading from a |previousversion| or
-        earlier server. If upgrading from a |version| server, you do not need
+        This section only concerns users upgrading from a 5.3 or
+        earlier server. If upgrading from a 5.4 or 5.5 server, you do not need
```

Comment:
Doesn't seem to be included in staged docs, guess not considered a point release.

Reply:
> Doesn't seem to be included in staged docs

Because of the job, right?

> guess not considered a point release.

What do you mean?

Reply:
I think the `.. only:: point_release` omits the warning altogether from staging.

Reply:
Gotcha. So the question might be how to reproduce the version that's on the website.